.github/workflows/compile-with-typescript-v4.yml

```diff
@@ -0,0 +1,22 @@
+name: Compile with typescript v4
+
+on:
+  push:
+    branches: main
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  compile_with_typescript_v4:
+    name: Compile with typescript v4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install typescript v4 in all packages
+        run: sed -i 's/"typescript": "~5.0.0"/"typescript": "^4.0.0"/' package.json packages/*/package.json && yarn
```

Comment:
don't you need two stars in the pattern `packages/*/package.json`?

Reply:
I don't think it's needed, as every package is right in `packages/`, like `packages/asd/package.json`